TextFieldEffects/TextFieldEffects/HoshiTextField.swift

```diff
@@ -143,15 +145,18 @@ typealias VoidClosure = () -> Void
                        height: height)
         configurePlaceholderLabelFrame()
         configurePlaceholderFont()
+	configureErrorLabel()
```

Comment:
is the spacing off in this and the next 2 addishes?

Reply:
yeah I dont know what is happening with indentation. It was good on Xcode but wrong on github, then i fixed it for github file view but it still looks wrong is file comparison view. So if you click on the button to view the full file (not just changes) it's right..